Index: b/arch/arm64/boot/dts/allwinner/sun50i-h616-orangepi-zero2.dts
===================================================================
--- a/arch/arm64/boot/dts/allwinner/sun50i-h616-orangepi-zero2.dts
+++ b/arch/arm64/boot/dts/allwinner/sun50i-h616-orangepi-zero2.dts
@@ -10,6 +10,7 @@
 #include <dt-bindings/gpio/gpio.h>
 #include <dt-bindings/interrupt-controller/arm-gic.h>
 #include <dt-bindings/leds/common.h>
+#include "sun50i-h616-cpu-opp.dtsi"
 
 / {
 	model = "OrangePi Zero2";
@@ -76,15 +77,15 @@
 	status = "okay";
 };
 
-&r_rsb {
+&r_i2c {
 	status = "okay";
 
-	axp305: pmic@745 {
+	axp305: pmic@36 {
 		compatible = "x-powers,axp305", "x-powers,axp805",
 			     "x-powers,axp806";
 		interrupt-controller;
 		#interrupt-cells = <1>;
-		reg = <0x745>;
+		reg = <0x36>;
 
 		x-powers,self-working-mode;
 		vina-supply = <&reg_vcc5v>;
@@ -152,7 +153,7 @@
 			reg_dcdca: dcdca {
 				regulator-always-on;
 				regulator-min-microvolt = <810000>;
-				regulator-max-microvolt = <1100000>;
+				regulator-max-microvolt = <1160000>;
 				regulator-name = "vdd-cpu";
 			};
 
@@ -184,6 +185,10 @@
 	};
 };
 
+&cpu0 {
+        cpu-supply = <&reg_dcdca>;
+};
+
 &pio {
 	vcc-pc-supply = <&reg_aldo1>;
 	vcc-pf-supply = <&reg_aldo1>;
@@ -193,7 +198,7 @@
 };
 
 &spi0  {
-	status = "okay";
+	status = "disabled";
 	pinctrl-names = "default";
 	pinctrl-0 = <&spi0_pins>, <&spi0_cs0_pin>;
 
